README.md

```diff
@@ -78,18 +78,19 @@ mvn clean package -Drevision=$(git describe --tags --always)
-| `aws.eventbridge.iam.external.id`    | No       |                            | The IAM external id (optional) when role-based authentication is used.                                                                                                                                                                                                                                                                                     |
+| Property                               | Required | Default                    | Description                                                                                                                                                                                                                                                                                                                                                  |
+|----------------------------------------|----------|----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `aws.eventbridge.connector.id`         | **Yes**  |                            | The unique ID of this connector (used in the EventBridge event `source` field as a suffix on `kafka-connect.` to uniquely identify a connector).                                                                                                                                                                                                             |
+| `aws.eventbridge.region`               | **Yes**  |                            | The AWS region of the target event bus.                                                                                                                                                                                                                                                                                                                      |
+| `aws.eventbridge.endpoint.uri`         | No       |                            | An optional [service endpoint](https://docs.aws.amazon.com/general/latest/gr/ev.html) URI used to connect to EventBridge.                                                                                                                                                                                                                                    |
```

Comment:
Endpoint.uri and Endpoint.id sound ambiguous. What about _aws.eventbridge.service.endpoint.uri_ and _aws.eventbridge.eventbus.global.endpoint.id_ ?

Reply:
Well, I thought about this one too. But note the difference between the prefixes `aws.eventbridge` and `aws.eventbridge.eventbus` (scoped to the bus). This would be a breaking change now (for `endpoint.uri`).

Reply:
Would you be ok with me making the `aws.eventbridge.eventbus.global.endpoint.id` change then for clarity?

Reply:
Yes then lets go with this one.